voorhoede/head-start
# feature: ActionBlock with InlineLink

# Changes

- Adds an Action Block model & component that can be used to add one or more prestyled actions (only Inline Link for now) to pages or other content blocks.
- Fixes the `npm create:...` (Plop) files so that test files are correctly added when creating a new Block or Component. (Not technically part of this PR, but I needed it to create the new blocks anyway).

```
┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄
┆ ╔═════════════╗  ┌─────────────┐            ┆
┆ ║   Primary   ║  │  Secondary  │   Default  ┆  = action group
┆ ╚═════════════╝  └─────────────┘   ───────  ┆
┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄┄
```

# Associated issue

Resolves #218

# How to test

1. Open preview link
2. Verify the home page has an ActionBlock above the fold and that it works
3. Navigate to the ActionBlock Demo page (`/en/demos/action-block-demo/`)
4. Verify that all the variations work

# Checklist

- ~I have added a decision log entry if the change affects the architecture or changes a significant technology~
